pom.xml

```diff
@@ -77,10 +77,9 @@
 
     <dependencies>
         <dependency>
-            <groupId>io.dropwizard</groupId>
-            <artifactId>dropwizard-metrics</artifactId>
-            <version>0.9.2</version>
-            <scope>provided</scope>
+            <groupId>io.dropwizard.metrics</groupId>
+            <artifactId>metrics-core</artifactId>
+            <version>3.1.2</version>
```

Comment:
Using the `provided` scope should be fine, although I'm not sure that would prevent overriding an older version in a project which is using `kafka-dropwizard-reporter` as a dependency.